packages/analytics/analytics-chart/src/components/AnalyticsChart.vue

```diff
@@ -190,6 +237,9 @@ const emit = defineEmits<{
 }>()
 
 const { i18n } = composables.useI18n()
+const { evaluateFeatureFlag } = composables.useEvaluateFeatureFlag()
+
+const hasKebabMenuAccess = computed(() => evaluateFeatureFlag('ma-3043-analytics-chart-kebab-menu', false))
```

Comment:
FYI, I don't think you need the `computed` here -- there's nothing reactive and the lookup should be cheap.